dashboard/app/controllers/activities_controller.rb

```diff
@@ -131,16 +131,6 @@ def milestone
       user_level: @user_level
     )
 
-    if solved
```

Comment:
Just curious what this was for?

Reply:
This was for the "activity monitor".  See this [slack thread](https://codedotorg.slack.com/archives/C0T0PNTM3/p1635530518124200) for more info on why it's no longer needed.